packages/camera/camera_avfoundation/example/ios/RunnerTests/Mocks/MockFrameRateRange.swift

```diff
@@ -0,0 +1,15 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// A mock implementation of `FLTFrameRateRange` that allows mocking the class properties.
+class MockFrameRateRange: NSObject, FLTFrameRateRange {
```

Comment:
final

Reply:
curious can this be just a struct? or does it have to be a nsobject subclass?

Reply:
I don't think it can without changing the definition of `@protocol FLTFrameRateRange <NSObject>`. I would hold off with this until `FLTFrameRateRange` is migrated to swift